pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
     "wheel>=0.29.0",
     "setuptools>=42.0.0",
-    "oldest-supported-numpy",
-    "Cython<3.0",
+    "numpy",
```

Comment:
Is it recommended to use "numpy" instead of "oldest-supported-numpy" now?

Reply:
I am not sure if that is recommended, but the problem is this:

requirements.txt and setup.py have just "numpy".  So, `pip install -r requirements.txt` installs the latest numpy. 

Then with `pip install -e .`  cyarray is compiled with oldest-supported-numpy as specified in pyproject.toml. 

Eventually it ends up with, 
```
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
``` 
like here: https://github.com/pypr/cyarray/actions/runs/10434451995/job/28897324886

Reply:
Hmm, so what is the oldest-supported-numpy version here? Is it < 2.0?

Reply:
Yes, its 1.23.2.